core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -29,24 +29,26 @@ function toTitleCase(str) {
 }
 
 @Component({
-    selector: 'dotcms-block-editor',
+    selector: 'dotcms-block-editor' /* eslint-disable-line */,
```

Comment:
?

Reply:
during build, require the prefix `dot-`,